packages/grid/tsconfig.build.json

```diff
@@ -5,6 +5,7 @@
     "declarationDir": "./dist",
     "module": "es6",
     "noImplicitAny": false,
+    "noUnusedLocals": false,
```

Comment:
```suggestion
```

Note that I normalize the tsconfig in https://github.com/mui-org/material-ui-x/pull/448.